source/containers/FormField/FormField.js

```diff
@@ -128,6 +128,7 @@ const inputTypes = {
     focusEvent: 'onFocus',
     blurEvent: 'onBlur',
     changeEvent: 'onChange',
+    onAddAnswerEvent: 'onAddAnswer',
```

Comment:
Noticed now but this is using a different naming scheme than the rest. In that case it should be renamed to `addAnswerEvent`.